.gitignore

```diff
@@ -40,6 +40,9 @@ dump.rdb
 # Ignore Rubymine
 .idea
 
+# Ignore asdf file
+.tool-versions
```

Comment:
this you could add in a global gitignore

https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
thanks for the tip.